### PR TITLE
Release gapics

### PR DESCRIPTION
**Pull request by andreamlin**
Generated with https://github.com/googleapis/gapic-generator/pull/2131/ to fix error in LoggingClient (see fix at https://github.com/andreamlin/google-cloud-java/commit/386c987bf831c34377e08a3e2638eb5d5ff5385b)

Update google-common-protos to v 1.12.0.

Add google-cloud-tasks for the first time.

Remove LoggingSmokeTest.java, because the Logging smoke config has been removed from googleapis/googleapis, and the existing LoggingSmokeTest.java is incompatible with the current google-cloud-logging/src/main.

### CODE REVIEW

**Comment by andreamlin** [comment]
Appveyor is failing due (I'm guessing) to a Java 7 error, where Maven (also gradle fails too) can't fetch from the dependency host https://repo.maven.apache.org/maven2 because that site uses TLS v 1.2, but Java 7 is by default used to use TLS v 1.1.

**Comment by andreamlin** [comment]
PTAL

File: versions.txt [hunk 1]
```diff
@@ -125,8 +127,9 @@ google-cloud-pubsub:1.35.0:1.35.1-SNAPSHOT
 google-cloud-redis:0.53.0-alpha:0.53.1-alpha-SNAPSHOT
 google-cloud-resourcemanager:0.53.0-alpha:0.53.1-alpha-SNAPSHOT
 google-cloud-spanner:0.53.0-beta:0.53.1-beta-SNAPSHOT
-google-cloud-speech:0.53.0-beta:0.53.1-beta-SNAPSHOT
+google-cloud-speech:0.53.0-alpha:0.53.1-alpha-SNAPSHOT
```
Comment (garrettjonesgoogle) [inline versions.txt]: No! I bumped this to beta on purpose...

Comment (andreamlin) [reply versions.txt]: Fixed.

**Comment by andreamlin** [comment]
PTAL

**Pull request merged**